src/core/mapping/base_mapper.cc

```diff
@@ -208,6 +208,14 @@ void BaseMapper::select_task_options(const MapperContext ctx,
                                      const LegionTask& task,
                                      TaskOptions& output)
 {
+  for (uint32_t idx = 0; idx < task.regions.size(); ++idx) {
```

Comment:
Let's wrap this loop with a C macro, say `LEGATE_NO_COLLECTIVE_INSTANCE`, so we can build the code with the `control_replication` branch as well. in the worst case where we don't do perf testing with the `collective` branch, we will need to create packages without collective instances.

Reply:
sounds good